CONTRIBUTING.md

```diff
@@ -74,6 +74,20 @@ optimize:
   lazyPrebuild: false
 ```
 
+#### SSL
+
+When Kibana runs in development mode it will automatically use bundled SSL certificates. These certificates won't be trusted by your OS by default which will likely cause your browser to complain about the cert. You can deal with this in a few ways:
+
+ 1. Configure your OS to trust the cert:
```

Comment:
I wonder if we shouldn't change the order here, as users are likely to use the first solution. I think this order makes more sense:

- supply your own valid cert
- force your OS to trust the cert provided
- click through the warnings
- run without ssl